inc/ticket.class.php

```diff
@@ -320,7 +318,7 @@ public static function addHistoryOnAddGroup(CommonDBTM $item)
             return;
         }
 
-        $tickets_id = $item->input['id'];
+        $tickets_id = $item->input['id'] ?? $item->input['tickets_id'];
```

Comment:
`$tickets_id` takes the value of `$item->input[‘id’]` only if `$item` is a ticket. If it isn't and the key exists, it takes the value `$item->input[‘tickets_id’]`

We probably need something more precise like this ?! (not tested):
```suggestion
        if ($item instanceof Ticket) {
            $tickets_id = $item->input['id'];
        } elseif (isset($item->input['tickets_id'])) {
            $tickets_id = $item->input['tickets_id'];
        } else {
            return false;
        }
```